mistakedetection/src/ca/mcgill/sel/mistakedetection/MistakeDetection.java

```diff
@@ -166,6 +166,12 @@ public class MistakeDetection {
 
   /** The maximum limit after which a resolved mistake will be removed from student solution. */
   public static final int MAX_DETECTIONS_AFTER_RESOLUTION = 5;
+  
+  /** The maximum limit after which a Levenstein distance of 2 is considered correct. */
+  public static final int MAX_WORD_LENGTH = 5;
```

Comment:
This name gives the impression that no words can be longer than 5 characters. Consider renaming to `MIN_LONG_WORD_LENGTH`, ie, the minimum length a word must have to be considered a "long word," where the words by length are defined as

| Name | Word length |
|---|---|
| Single-letter word | 1 |
| Short word | 2-4 |
| Long word | 5+ |

Reply:
@Prabhsimran-Singh This can be done later